Makefile

```diff
@@ -142,14 +142,17 @@ push-local-images:
 	docker push localhost:5000/tarian-node-agent
 
 unit-test:
-	CGO_CFLAGS=$(CGO_CFLAGS_STATIC) CGO_LDFLAGS=$(CGO_LDFLAGS_STATIC) go test -v -race -count=1 ./pkg/...
```

Comment:
`-count=1` was used to avoid test cache. It forces the test to run even though the code doesn't change.

Reply:
Okay, I'll update it.